src/download.ts

```diff
@@ -159,13 +161,17 @@ export function getSkylinkUrl(this: SkynetClient, skylinkUrl: string, customOpti
   if (opts.download) {
     query.attachment = true;
   }
+  if (opts.noResponseMetadata) {
+    query["no-response-metadata"] = true;
```

Comment:
why are we deviating from the format here? `query.attachment` vs `query["no-response-metadata"]
shouldn't we use camel casing? `query.noResponseMetadata`?

Reply:
Well the parameter is `no-response-metadata`, camel casing wouldn't work.